test/parallel/test-fs-readfile-pipe-large.js

```diff
@@ -5,8 +5,8 @@ var path = require('path');
 
 // simulate `cat readfile.js | node readfile.js`
 
-if (common.isWindows) {
-  console.log('1..0 # Skipped: No /dev/stdin on windows.');
+if (common.isWindows || common.isAix) {
+  console.log('1..0 # Skipped: No /dev/stdin' + process.platform + '.');
```

Comment:
This will print `No /dev/stdinaix`.  Probably unintended.

You can write this using template strings, by the way, might be a little neater:

```javascript
console.log(`1..0 # Skipped: No /dev/stdin on ${process.platform}.`);
```